doc/amazon_sagemaker_operators_for_kubernetes_jobs.rst

```diff
@@ -304,6 +304,135 @@ job stops or completes.
 continue to show on the Amazon SageMaker console. The delete command
 takes about 2 minutes to clean up the resources from Amazon SageMaker.
 
+SageMaker Debugger Jobs
+^^^^^^^^^^^^^^^^^^^^^^^
+
+When creating a SageMaker training job, you have an option to run 
+asynchronous debugger jobs for your model. It gives you full visibility 
+into a training job by using a hook to capture tensors that define 
+the state of the training process at each instance in its lifecycle. 
+It also provides the capability of defining 'rules' to
+analyze the captured tensors. See \ `SageMaker Debugger Introduction <https://docs.aws.amazon.com/sagemaker/latest/dg/train-debugger.html>`__ and \ `How Debugger Works <https://docs.aws.amazon.com/sagemaker/latest/dg/debugger-how-it-works.html>`__ for details.
+
+You can get more details on debug job by using ``describe`` kubectl verb.
```

Comment:
* s/debug/a debugging (or do you mean a training job here?)
* s/using/using the

Reply:
It is a trainingjob which has `side job` (debug job).

Reply:
gotcha. it might help to describe that since right now it reads "you can describe the debug job. here's a command that says training job" 😂

Reply:
Well, debug jobs lives inside the training job not separately so unfortunately I am stuck with the same command. :-/

Reply:
I think you could just rephrase it to something like "You can get more details on a training job's debugging job..."

Reply:
At the beginning of paragraph its clear that debug is being launched as part of training job so don't know if I should really make that change
`When creating a SageMaker training job, you have an option to run 
asynchronous debugger jobs for your model...`